lib/scss_lint/linter/use_variables_for_properties.rb

```diff
@@ -0,0 +1,18 @@
+module SCSSLint
+  class Linter::UseVariablesForProperties < Linter
+    include LinterRegistry
+
+    def visit_root(_node)
+      @included_properties = Set.new(config['included_properties'])
+      yield
+    end
+
+    def visit_prop(node) 
+      return if node.name.count > 1 || !node.name.first.is_a?(String)
+      return unless @included_properties.include? node.name.first
+      return if node.children.first.is_a?(Sass::Script::Tree::Variable)
```

Comment:
That's a great point. How do you feel about leaving it to variables for now and then figuring out the function thing at some other time? Would that cause trouble by possibly breaking your API?